secure_cloudfront_video/views.py

```diff
@@ -32,6 +34,9 @@ def secure_cloudfront_video(request):
         Redirect 302: If the signing process was successful and the resource exists.
         Not Found 404: If the 'key' query string or any of the Amazon Cloud Front settings is missing.
     """
+    if not is_request_from_mobile_app(request) and not request.user.is_authenticated:
+        raise Http404
```

Comment:
```suggestion
        raise Http404('Authentication required for web requests.')
```

Reply:
I don't think we should add this as we don't want any request to know the endpoint actually exists unless there is a legitimate request, that's why other 404s responses does not lead the request to anything and just return 404. However I added some logging for us to use it in case we need to debug. @anfbermudezme